crypto/err/err.c

```diff
@@ -23,6 +23,24 @@
 #include "internal/constant_time_locl.h"
 #include "e_os.h"
 
+#define ERR_FLAG_MARK      0x01
+#define ERR_FLAG_CLEAR     0x02
+#define ERR_NUM_ERRORS     16
+
+typedef struct err_entry_st {
+    int err_flags;
+    unsigned long err_buffer;
+    char *err_data;
+    int err_data_flags;
+    const char *err_file;
+    int err_line;
+} ERR_ENTRY;
+
+typedef struct err_state_st {
```

Comment:
It  needs to just be
struct err_state_st {
}; /* ERR_STATE */

This is one of my least favorite compiler errors :)